miner/worker.go

```diff
@@ -55,6 +55,9 @@ var (
 	errBlockInterruptedByTimeout  = errors.New("timeout while building block")
 	errBlockInterruptedByResolve  = errors.New("payload resolution while building block")
 
+	// OP-Stack addition
+	errSupervisorInFailsafe = errors.New("supervisor in failsafe")
+
```

Comment:
can we use the other error you already defined in the interop package?

Reply:
I don't think so. They are two different types of error: this one is local to op-geth and returned by `miner.commitTransaction`. The other one is defined in op-supervisor and duplicated inside op-geth so that it can pattern match against it (i.e. detect that particular error).